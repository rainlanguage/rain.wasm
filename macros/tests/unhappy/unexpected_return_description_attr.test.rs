#[macro_use]
extern crate wasm_bindgen_utils_macros;

struct TestStruct;

#[wasm_export(return_description = "unexpected description")]
impl TestStruct {
    pub async fn some_method(arg: String) -> Result<String, Error> {
        Ok(String::new())
    }
}

fn main() {}