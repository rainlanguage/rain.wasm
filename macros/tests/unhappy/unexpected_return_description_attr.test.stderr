error: unexpected `return_description` attribute, it can only be used for impl block methods or standalone functions
 --> tests/unhappy/unexpected_return_description_attr.test.rs:6:15
  |
6 | #[wasm_export(return_description = "unexpected description")]
  |               ^^^^^^^^^^^^^^^^^^
